src/medley/core.cljc

```diff
@@ -506,6 +506,42 @@
   ([pred coll]
    (partition-between (fn [_ x] (pred x)) coll)))
 
+(defn window
+  "A sliding window, returning a lazy sequence of partitions, containing each
+  element and n-1 preceeding elements, when present. Therefore partitions at the
+  start may contain fewer items than the rest. Returns a stateful transducer
+  when no collection is provided. For a sliding window containing each element
+  and n-1 _following_ elements, use `clojure.core/partition` with a `step` size
+  of 1."
+  {:added "1.10.0"}
```

Comment:
Should be 1.9.0 I believe.

Reply:
Yes, sorry I assumed 1.9 had been cut. Will correct.